sherpa/models/tests/test_regrid_unit.py

```diff
@@ -437,7 +437,7 @@ def test_regrid1d_error_grid_mismatch_1(setup_1d):
 
     assert ModelErr.dict['needsint'] in str(excinfo.value)
 
-def test_regrid1d_non_overlapping_not_allowed():
+def test_ui_regrid1d_non_overlapping_not_allowed():
```

Comment:
Now that the check has been moved to the `regrid` method, there really isn't any difference between `test_ui_regrid1d_non_overlapping_not_allowed` and `test_low_level_regrif1d_non_overlapping_not_allowed`, since both are really just

  - create model instance
  - call regrid on that model instance, check for it raising an error

and the rest of the test (e.g. calling `ui.set_model` or creating a `Data1DInt` instance) isn't adding anything.